src/js/providers/utils/stream-type.ts

```diff
@@ -1,34 +1,35 @@
 /** @module */
+import { StreamType } from 'types/generic.type';
```

Comment:
given it's part of this file, I think it makes more sense to export it from this file. the generic is mostly used for placeholder types we hope to fill in later